src/core/Akka.Remote/Transport/TestTransport.cs

```diff
@@ -51,10 +54,11 @@ private static string GetConfigString(Config conf, string name)
             return value;
         }
 
-        public TestTransport(Address localAddress, AssociationRegistry registry, string schemeIdentifier = "test")
+        public TestTransport(Address localAddress, AssociationRegistry registry, long maximumPayloadBytes = 32000, string schemeIdentifier = "test")
```

Comment:
Does the Scala JVM transport support this constructor out of the box? Been a while since I've looked